src/main/java/org/pac4j/vertx/handler/impl/RequiresAuthenticationHandler.java

```diff
@@ -219,7 +219,7 @@ protected void redirectToIdentityProvider(final VertxWebContext webContext, fina
         }
     }
 
-    protected void unauthorized(final VertxWebContext webContext, List<Client> currentClients) {
```

Comment:
The WebContext is included to enable override of the behaviour based on the request. I don't think we should remove it as a parameter from this method (any comment Jérôme?)